code/go/0chain.net/miner/block_notarization_entity_test.go

```diff
@@ -0,0 +1,56 @@
+package miner
+
+import (
+	"0chain.net/chaincore/block"
+	"0chain.net/core/mocks"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+type BlockNotarizationEntityTestSuite struct {
+	suite.Suite
+}
+
+func TestBlockNotarizationEntityTestSuiteSuite(t *testing.T) {
```

Comment:
You don't seem to actually be testing anything in these tests.

Proably a key point here is that `Notarization` satisfies both the `Entity` and `ReadLockable` interfaces.

You can test DoReadLock and DoReadUnlock by calling various read and write calls on the `Entity` interface in Parallel. Then run `go test -race` to show there are no concurrency errors.